src/cobra/core/model.py

```diff
@@ -358,7 +362,7 @@ def copy(self):
             new.genes.append(new_gene)
 
         new.reactions = DictList()
-        do_not_copy_by_ref = {"_model", "_metabolites", "_genes"}
+        do_not_copy_by_ref = {"_model", "_metabolites", "_genes", "_gpr"}
```

Comment:
not necessary, was already fixed in the previous PR. The default copy on a ref now works for the GPR class.

Reply:
Right, got it.